app/code/Magento/Sales/Test/Unit/ViewModel/CreditMemo/Create/ItemsToRenderTest.php

```diff
@@ -126,6 +126,10 @@ public function testGetItems(): void
             ->willReturn($this->creditmemo);
         $this->creditmemo->method('getStoreId')
             ->willReturn(1);
+        $this->orderItem->method('getQtyInvoiced')
+            ->willReturn($this->orderItem);
+        $this->orderItem->method('getQtyRefunded')
+            ->willReturn($this->orderItem);
```

Comment:
@engcom-Delta, I think it's not correct code. It should return the qty, not the quote item object